graphbolt/src/fused_csc_sampling_graph.cc

```diff
@@ -337,7 +338,7 @@ auto GetNumPickFn(
   // If fanouts.size() > 1, returns the total number of all edge types of the
   // given node.
   return [&fanouts, replace, &probs_or_mask, &type_per_edge](
-             int64_t offset, int64_t num_neighbors) {
+             int64_t seed_offset, int64_t offset, int64_t num_neighbors) {
```

Comment:
Is this used to get the other attribute like timestamp of the seed node?

Reply:
Yes. We can use the seed_offset to get both the seed id and seed timestamp, i.e., `input_nodes[seed_offset]` and `input_nodes_timestamp[seed_offset]`.